src/plugin.ts

```diff
@@ -46,7 +46,7 @@ export default function makePlugin(config: Config = {}): Plugin {
     }
   }
 
-  const sourceDirectory = path.resolve(cwd, (relayConfig['src'] as string) || './src');
+  const sourceDirectory = path.resolve(cwd, (relayConfig['src'] as string) || './src').split(path.sep).join(path.posix.sep);
```

Comment:
Can we use `path.posix.resolve` or `path.posix.join` here instead?

Reply:
That's what I've mentioned; I've tried that but it doesn't return the drive letter at front of the path, therefore a mismatch between the `id` and the actual path still occurs.

Reply:
damn